content/2016/09/06/css-grid-layout.rst

```diff
@@ -0,0 +1,183 @@
+
+
+Grid introduces new vocabulary and CSS properties
+-------------------------------------------------
+
+New grid terminology includes:
+
+.. line-block::
+
+  **Grid line**
+  The lines that create the grid, separating the grid cells.
+
+  **Grid track**
+  The horizontal or vertical space between two grid lines, often spanning multiple grid cells.
+
+  **Grid cell**
+  A single unit of the grid made from the space between four grid lines.
+
+  **Grid area**
+  A group of space between four grid lines, often containing a group of grid cells. Grid areas can be named in CSS.
+
+
+CSS Grid Layout Properties you would use on a grid container:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  - display
+  - grid-template-columns
+  - grid-template-rows
+  - grid-template-areas
+  - grid-column-gap
+  - grid-row-gap
+  - grid-gap
+  - justify-items
+  - align-items
+  - justify-content
+  - align-content
+  - grid-auto-columns
+  - grid-auto-rows
+  - grid-auto-flow
+  - grid
+
+
+**CSS Grid Layout Properties you would use on a grid item:**
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  - grid-column-start
+  - grid-column-end
+  - grid-row-start
+  - grid-row-end
+  - grid-column
+  - grid-row
+  - grid-area
+  - justify-self
+  - align-self
+
+
+`CSS-Tricks posted a guide`_ from Chris House `Chris House`_ that goes in depth into each of these properties.
+
+.. _CSS-Tricks posted a guide: https://css-tricks.com/snippets/css/complete-guide-grid
+.. _Chris House: http://chris.house/blog/a-complete-guide-css-grid-layout/
+
+
+I’ve been creating a few demos in CodePen using Grid and it has been exciting to see the flexibility we will have once this rolls out. I encourage you to start experimenting on your own as well.
+
+
+.. raw:: html
+
+  <p data-height="568" data-theme-id="21914" data-slug-hash="rLyErg" data-default-tab="result" data-user="stacy" data-embed-version="2" class="codepen">See the Pen <a href="http://codepen.io/stacy/pen/rLyErg/">CSS Grid Layout Demo</a> by Stacy (<a href="http://codepen.io/stacy">@stacy</a>) on <a href="http://codepen.io">CodePen</a>.</p><script async src="//assets.codepen.io/assets/embed/ei.js"></script>
+
+
+What types of layouts can we create with this more flexible system? I would love for some very experimental design to start taking place. I can hear the sighs from usability experts everywhere so let me be clear, I am not saying that we need to create crazy, chaotic designs with unpredictable navigation patterns. I am only asking how we can explore and create new ways to layout out content that are still intuitive but perhaps different from what we've always done in the past.
+
+Flexbox and CSS Grid Layout Resources
```

Comment:
I love having all these resources available. When I got down to the conference videos, though, I wasn't sure about the context. I thought maybe they were videos of you? Maybe add some intro text there?